### PR TITLE
Add script to automate submission to VirusTotal and MetaDefender

### PR DESCRIPTION
**Pull request by EchterAgo**
For the script to work you need to set your API keys in the VIRUSTOTAL_API_KEY and METADEFENDER_API_KEY environment variables.

Just call like this, for example:

```
./contrib/submit_av.sh dist/*.exe
```

### CODE REVIEW

**Comment by cculianu** [comment]
This is great man. THANK YOU!

**Comment by SomberNight** [comment]
Could you give some context for this? What is the motivation?

**Comment by cculianu** [comment]
This: https://old.reddit.com/r/btc/comments/bwaasu/electron_cash_windows_405_exe_trojan_warning/
This: https://www.virustotal.com/gui/file/7dd97481328132289e832d7f0881c8c4ac0072850d8e5e617ba1ff7d355b8352/detection

Electrum affected too, BTW: https://www.virustotal.com/gui/file-analysis/NDA5NWFkZDE5ZjQwNzU1ODMzMDM1YjFlMzFhNjU4M2U6MTU1OTU3Mjc4MQ==/detection

**tl;dr** Virus companies are very very wrong sometimes.. :)  It's best to submit the binary early to them and then either see if the detection is real (unlikely, given how we do build) or work with them to whitelist it if it's a false positive (most of the time this is the end result).

Also if they do a detection early they also have time to look at the file and whitelist it later  -- thus making sure the window of time where users get a false detection is as small as possible (these false detections often go away on their own as binaries make it to the virus companies).

So.. by submitting early to them -- the binaries get there faster, and thus get whitelisted faster.
